### ISSUE TITLE
FAILED: py27.test.reflection._reflection_servicer_test.ReflectionServicerTest

### ISSUE THREAD

**Issue by apolcyn** [grpc/grpc#13516]
https://sponge.corp.google.com/invocation?tab=Kokoro&id=fd975818-6d57-4398-84d3-746e17867e98&searchFor=

Looks like main error here is `AttributeError: 'NoneType' object has no attribute 'CopyToProto'
`

```
SUCCESS       reflection._reflection_servicer_test.ReflectionServicerTest.testListServices
5 tests finished:
	4 successful
	1 unsuccessful
	0 skipped
	0 expected failures
	0 unexpected successes
Interrupted Tests:
	[]

Errors/Failures: 
reflection._reflection_servicer_test.ReflectionServicerTest.testFileContainingExtension
traceback:
Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.13_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/Volumes/BuildData/tmpfs/src/github/grpc/workspace_python_macos_dbg_native/src/python/grpcio_tests/tests/reflection/_reflection_servicer_test.py", line 107, in testFileContainingExtension
    responses = tuple(self._stub.ServerReflectionInfo(iter(requests)))
  File "/Volumes/BuildData/tmpfs/src/github/grpc/workspace_python_macos_dbg_native/py27/lib/python2.7/site-packages/grpc/_channel.py", line 351, in next
    return self._next()
  File "/Volumes/BuildData/tmpfs/src/github/grpc/workspace_python_macos_dbg_native/py27/lib/python2.7/site-packages/grpc/_channel.py", line 342, in _next
    raise self
_Rendezvous: <_Rendezvous of RPC that terminated with (StatusCode.UNKNOWN, Exception iterating responses: 'NoneType' object has no attribute 'CopyToProto')>

stdout:

stderr:
ERROR:root:Exception iterating responses: 'NoneType' object has no attribute 'CopyToProto'
Traceback (most recent call last):
  File "/Volumes/BuildData/tmpfs/src/github/grpc/workspace_python_macos_dbg_native/py27/lib/python2.7/site-packages/grpc/_server.py", line 389, in _take_response_from_response_iterator
    return next(response_iterator), True
  File "/Volumes/BuildData/tmpfs/src/github/grpc/workspace_python_macos_dbg_native/py27/lib/python2.7/site-packages/grpc_reflection/v1alpha/reflection.py", line 116, in ServerReflectionInfo
    request.file_containing_extension.extension_number)
  File "/Volumes/BuildData/tmpfs/src/github/grpc/workspace_python_macos_dbg_native/py27/lib/python2.7/site-packages/grpc_reflection/v1alpha/reflection.py", line 81, in _file_containing_extension
    return _file_descriptor_response(descriptor)
  File "/Volumes/BuildData/tmpfs/src/github/grpc/workspace_python_macos_dbg_native/py27/lib/python2.7/site-packages/grpc_reflection/v1alpha/reflection.py", line 35, in _file_descriptor_response
    descriptor.CopyToProto(proto)
AttributeError: 'NoneType' object has no attribute 'CopyToProto'


Unexpected successes: []
Test failure

2017-11-17 22:05:31,077 FAILED: py27.test.reflection._reflection_servicer_test.ReflectionServicerTest [ret=1, pid=47306, time=1.5sec]
```

-----

also seen on windows in https://sponge.corp.google.com/target?id=6988f3fa-75ce-4d6c-b49c-8e4ac1e23b1b&target=github/grpc/python_windows_dbg_native&searchFor=&show=FAILED&sortBy=STATUS

**Comment by mehrdada** [comment]
This should have been fixed with the protobuf `3.5.0.post1` release. Could it be that the worker is using an old copy of protobuf?

**Comment by apolcyn** [comment]
this should be fixed by recent PR since this test run
